cmeutils/plotting.py

```diff
@@ -0,0 +1,36 @@
+import numpy as np
+
+
+def get_histogram(data, normalize=False, bins="auto"):
+    """Bins a 1-D array of data into a histogram using
+    the numpy.histogram method.
+
+    Parameters
+    ----------
+    data : 1-D numpy.array, required
+        Array of data used to generate the histogram
+    normalize : boolean, default=False
+        If set to true, normalizes the histogram bin heights
+        by the sum of data so that the distribution adds
+        up to 1
+    bins : float, int, or str, default="auto"
+        Method used by numpy to determine bin borders.
+        Check the numpy.histogram docs for more details.
+
+    Returns
+    -------
+    bin_cetners : 1-D numpy.array
+        Array of the bin center values
+    bin_heights : 1-D numpy.array
+        Array of the bin height values
+
+    """
+    bin_heights, bin_borders = np.histogram(data, bins=bins)
+    if normalize is True:
+        s = sum(bin_heights)
+        bin_heights = np.array(
+                [float(i)/s for i in bin_heights]
+        )
```

Comment:
```suggestion
        bin_heights /= sum(bin_heights)
```

Reply:
This throws an error.

```
numpy.core._exceptions._UFuncOutputCastingError: Cannot cast ufunc 'true_divide' output from dtype('float64') to dtype('int64') with casting rule 'same_kind'
```

I did change it so it's a bit cleaner/faster though.